pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -55,6 +55,8 @@ func init() {
 			rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams").RuleOrDie(),
 			rbac.NewRule("get", "list", "create", "delete").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 			rbac.NewRule("get").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("secrets").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("configmaps").RuleOrDie(),
```

Comment:
In the other pull for the build controller I am also adding secrets (let's see which one lands first :-) )

@jim-minter this is a service account that regular users normally don't have access to (in `openshift-infra`), and where other sa's with more power also live. So I don't think it adds any more security exposure.